server.py

```diff
@@ -7,13 +6,27 @@
 
 import flask as flask
 
+
+# ------------------------------------------------------------------------------
+# Flask app
+# ------------------------------------------------------------------------------
+
 app = flask.Flask("crisis", static_folder="build/", static_url_path="/")
 logger = app.logger
 
+# In development mode, allow access from the Yarn-hosted frontend.
+if os.environ.get("FLASK_ENV") == "development":
```

Comment:
Guessing this is set through `.flaskenv`, does this mean the server now _must_ be run using `flask run`?

Reply:
For running a development instance, yes. It's the recommended way because it's the only reliable way to get automatic reloading when the source code changes. (https://flask.palletsprojects.com/en/1.1.x/server/) If you think that's a problem, I guess we could add the main() function back and support both methods, it just seemed better to avoid having the code if it wasn't needed.

Reply:
This seems fine, was just clarifying.